app/api/auth/login/gallery/route.ts

```diff
@@ -0,0 +1,43 @@
+import { ConflictError } from "@/custom/errors/dictionary/errorDictionary";
+import { handleErrorEdgeCases } from "@/custom/errors/handler/errorHandler";
+import { connectMongoDB } from "@/lib/mongo_connect/mongoConnect";
+import {
+  AccountGallery,
+  type GallerySchemaTypes,
+} from "@/models/auth/GallerySchema";
+import bcrypt from "bcrypt";
+import { NextResponse as res } from "next/server";
+
+export async function POST(request: Request) {
+  try {
+    await connectMongoDB();
+
+    const data = await request.json();
+
+    const { email, password } = data;
+
+    const user = await AccountGallery.findOne<GallerySchemaTypes>(
+      { email },
+      {
```

Comment:
Please re-format this line to be:
const user = await AccountGallery.findOne<GallerySchemaTypes>({email}, "gallery_id, password"})

This is the recommendation from the mongoose documentation

Reply:
oh, i didn't know you can do that

Reply:
can you provide the link

Reply:
![mongoose](https://github.com/Moses-py/omenai/assets/44558083/f5dfd5e8-6328-4a1a-a2e9-33d1b26a5365)

Reply:
https://mongoosejs.com/docs/api/model.html#Model.findOne()